### ISSUE TITLE
sbt-vamp

### ISSUE THREAD

**Issue by plamola** [magneticio/vamp#280]
**tldr; Create a SBT plugin which can talk to VAMP.**

We currently use SBT plugins like sbt-bintray & sbt-native-packager, combined with Travis-ci, to automate the release part of our development workflow. This can all be controlled from within SBT (and some in travis.yml). 

We should enable other developers to automate their work as well. In part, this can already be done using Vamp CLI, but there still is the need to write scripting, to link it all together. For an easier adoption, a SBT plugin could be created, which has much of the same functionality as Vamp CLI. A developer could use a command like 
```
sbt vamp:deploy
```
to deploy a build artifact to vamp.

## Example 
A developer wants to run a new docker image:
```
sbt clean test docker:publishLocal
docker push mynewimage:latest
sbt vamp:deploy mynewimage:latest
```
